usr/src/test/zfs-tests/include/libtest.shlib

```diff
@@ -1675,6 +1675,26 @@ function is_pool_removed #pool
 	return $?
 }
 
+function wait_for_degraded
+{
+	typeset pool=$1
+	typeset timeout=${2:-30}
+	typeset t0=$SECONDS
+
+	while :; do
+		zpool status -v $pool 2>/dev/null | grep -q "state: DEGRADED"
```

Comment:
Using ```zpool get -Ho value health``` would look somewhat cleaner and less susceptible to output format changes.